index.js

```diff
@@ -50,6 +50,18 @@ MongoClient.connect('mongodb://localhost/notebook', (err, db) => {
       })
   })
 
+  app.delete('/api/notes/:id', (req, res) => {
+    notes
+      .deleteOne({ id: req.params.id }, (err, result) => {
+        if (err) {
+          console.error(err)
+          res.sendStatus(500)
+        }
+        console.log(req.body)
+        res.sendStatus(202)
```

Comment:
You want `204 "No Content"`

Reply:
@thebearingedge Good call. Thanks!

Fixed.